sdk/src/main/java/io/radar/sdk/model/RadarMeta.kt

```diff
@@ -0,0 +1,31 @@
+package io.radar.sdk.model
+
+import io.radar.sdk.RadarTrackingOptions
+import org.json.JSONObject
+
+/**
+ * Defines Metadata that is used by the Radar SDK
+ */
+internal data class RadarMeta(
+    val config: JSONObject?,
+    val remoteTrackingOptions: RadarTrackingOptions?
+) {
+    companion object {
+        private const val META = "meta"
+        private const val CONFIG = "config"
+        private const val TRACKING_OPTIONS = "trackingOptions"
+
+        fun fromJson(res: JSONObject?): RadarMeta {
+            val meta = res?.optJSONObject(META)
+            val config = meta?.optJSONObject(CONFIG)
```

Comment:
In the discussion of the iOS PR, it turns out that `trackingOptions` supersedes the `config` parameter, so should it be dropped here, too?

Reply:
let's just kill it, less code is better

Reply:
Adding reference for visibility https://github.com/radarlabs/radar-sdk-ios/pull/127

Reply:
@jrtibbetts can you link to the specific comment you mean? Trying to determine the ask here - to delete the config field?

Reply:
Removed.